src/devices/Card/Icode/IcodeCardCommand.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Card.Icode
+{
+    /// <summary>
+    /// List of commands available for the Icode cards
+    /// </summary>
+    public enum IcodeCardCommand
```

Comment:
Consider making this internal and only expose public methods on the Icode* class